app/views/news/2018-09-06-august-2018-monthly-update.html.md

```diff
@@ -0,0 +1,72 @@
+---
+summary: In August, we paid for 113 hours of developer work, shipped a new Bundler guide, and fixed bugs in Bundler and RubyGems.
+---
+
+Hello! Welcome to the monthly update. During August, our work was supported by [Handshake](https://handshake.org), [reinteractive](https://reinteractive.com/), and many others.
+
+## ruby together news
+
+In August, Ruby Together was supported by 74 different companies. [Handshake](https://www.handshake.org) joined as a [LEVEL] member. In total, we were supported by 77 individual members and 64 friends of Ruby Together. Thanks to all of our members for making everything that we do possible.
+
+In community events, our developer evangelist PJ Hagerty spent some time at the Google Developer Group meetup in New York City, Scenic City Summit in Chattanooga, TN, and AmsterdamRB. Executive director André Arko went to RailsCamp to spread the word about Ruby Together. 
+
+In other community news, our Google Summer of Code students wrapped up their work on our various projects. Special thanks to our GSoC developers for all of their hard work these past few months!
+
+## bundler news
+
+And with the help of [@eanlain](github.com/eanlain), we shipped a new guide: [“How to use Bundler with Docker”](https://bundler.io/v1.16/guides/bundler_docker_guide.html).
```

Comment:
> bundler news

@indirect I imagine there were way more things that happened in Bundler aside from the Docker guide. I structured it so we'd open with whatever important changes happened last month, and they'd end with the guide and w/the announcement of me joining the core team. Let me know if that works!